apps/web/vibes/soul/sections/blog-post-content/index.tsx

```diff
@@ -36,33 +37,24 @@ export const BlogPostContent = function BlogPostContent({
   content,
   image,
   className = '',
+  breadcrumbs,
 }: Props) {
   return (
     <section className={clsx('@container', className)}>
       <div className="mx-auto max-w-screen-2xl px-4 py-10 @xl:px-6 @xl:py-14 @4xl:px-8 @4xl:py-20">
         <header className="mx-auto w-full max-w-4xl pb-8 @2xl:pb-12 @4xl:pb-16">
-          <Breadcrumbs
-            breadcrumbs={[
-              {
-                label: 'Home',
-                href: '#',
-              },
-              {
-                label: 'Blog',
-                href: '#',
-              },
-              {
-                label: title,
-                href: '#',
-              },
-            ]}
-          />
+          {breadcrumbs && <Breadcrumbs breadcrumbs={breadcrumbs} />}
 
           <h1 className="mb-4 mt-8 font-heading text-4xl font-medium leading-none @xl:text-5xl @4xl:text-6xl">
             {title}
           </h1>
           <p>
-            {date} <span className="px-1">•</span> {author}
+            {date}{' '}
+            {Boolean(author) && (
+              <>
+                <span className="px-1">•</span> {author}
+              </>
+            )}
```

Comment:
We seem to be rendering a space regardless of whether the author is present or not. We should move that space inside the conditional. Also, instead of coercing using the `Boolean` constructor, can we check `auth != null` to be consistent?